docs/metadata.md

```diff
@@ -22,378 +22,206 @@ searching for particular kinds of data.
 
-### text/csv
+[dcat:versionNotes](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_version_notes)
+: Notes used to explain any changes to this version.
 
-This type indicates that data is presented in CSV format. In this case, an optional key `csvw` may be defined, and 
-should contain valid [JSON-LD](glossary.md#term-JavaScript-Object-Notation-for-Linked-Data) following the [CSV for the Web](https://www.w3.org/TR/tabular-data-primer/) guidelines:
+[dcat:distribution](https://www.w3.org/TR/vocab-dcat-3/#Property:dataset_distribution)
+: URL of a `dcat:Distribution` for a downloadable file, see below for mandatory fields. Multiple Distribtions may be defined for the same data in different formats, but Scheme-conforming datasets will require a specific format.
 
-```yaml
-representation:
-  mime: text/csv
-  csvw:
-    # This is only applicable if the mime type is text/csv
-    "@context": http://www.w3.org/ns/csvw
-    tableSchema:
-      columns:
-        - titles: country
-        - titles: country group
-        - titles: name (en)
-        - titles: name (fr)
-        - titles: name (de)
-        - titles: latitude
-        - titles: longitude
-```
+[dcat:inSeries](https://www.w3.org/TR/vocab-dcat-3/#Property:dataset_in_series)
+: Where the URL is part of a series of periodic datasets, the URL of a `dcat:DataSeries` which associates this Dataset with the overall series. The DataSeries is created by the publisher and contains their data only.
 
-### Other types
+Any additional metadata defined by published Standards may be added.
+
+### Distribution metadata fields
+
+To specify how the data may be downloaded, one or more associated `dcat:Distribution` objects must be included which contain:
+
+[dcat:downloadURL](https://www.w3.org/TR/vocab-dcat-3/#Property:distribution_download_url)
+: A stable URL for download of the dataset, subject to access controls specified in the Dataset object. Liveness of the server will be tested by making a HEAD request to this URL.
+
+[dcat:media_type](https://www.w3.org/TR/vocab-dcat-3/#Property:distribution_media_type)
+: The MIME type of the download file.
+
+`ib1:dataSchema`
+: The URL of a schema file specifying the format of the downloadable file. The type of schema depends on the `dcat:mediaType`:
+`application/json` are documented by JSON Schema files,
+`application/xml` by XSD 1.1 files, and
+`text/csv` by [CSVW](https://www.w3.org/ns/csvw) files.
+
+### Additional metadata for Datasets and Data Services
+
+The information above is the minimum needed to ensure that a data source can be used by the Trust Framework participants, and is visible in [the Open Net Zero](https://opennetzero.org) search system. There 
```

Comment:
Some info above is optional (at least I think so). So maybe "The fields that are marked as mandatory are the minimum to ensure..."